src/components/Login.jsx

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+import auth from "../modules/auth";
+import { connect } from "react-redux";
+import { Button } from 'semantic-ui-react'
+
+const Login = props => {
+  const onLogin = event => {
+    event.preventDefault();
+    auth
+      .signIn(event.target.email.value, event.target.password.value)
+      .then(userDatas => {
+        props.changeAuth(true);
+        props.setUserAttrs(userDatas.data);
+        props.changeAuthMessage(`Logged in as: ${userDatas.data.email}`);
+      })
+      .catch(error => {
+        props.changeAuthMessage(error.response.data.errors);
+      });
+  };
+  const onLogout = () => {
+    auth
+      .signOut()
+      .then(() => {
+        props.changeAuth(false);
+        props.changeLoginButton(true);
+        props.changeAuthMessage("");
+      })
+      .catch(error => {
+        props.changeAuthMessage(error);
+      });
+  };
+  
+  let loginFunction;
+  switch (true) {
+    case props.displayLoginButton &&
+      !props.authenticated:
+      console.log("entered 0")
```

Comment:
You can remove these console.log statements

Reply:
done